docs/DeveloperGuide.md

```diff
@@ -1039,7 +1063,58 @@ testers are expected to do more *exploratory* testing.
    1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
       Expected: Similar to previous.
 
-1. _{ more test cases …​ }_
+### Adding a lesson
```

Comment:
Formatting for this portion is a bit too different from the other sections imo

Reply:
This part belongs to the Appendix which is an entire section on its own plus I didn't change the formatting, this was how it originally was in the template.